Modelica/Magnetic/QuasiStatic/FluxTubes/Shapes/FixedShape/HollowCylinderRadialFlux.mo

```diff
@@ -16,7 +16,7 @@ model HollowCylinderRadialFlux
   parameter SI.Angle alpha=2*Modelica.Constants.pi
   "Central angle";
 equation
-  A = l*pi*(r_o + r_i);
+  A = (r_o + r_i)/2*alpha*l;
```

Comment:
That is not good style - it's easy to think a/b*c means a/(b*c) instead of (a/b)*c.
```suggestion
  A = l*alpha*(r_o + r_i)/2;
```

Reply:
I agree with @HansOlsson that writing it this way is much cleaner.